DevAcademy/Scenes/Places/PlacesRow.swift

```diff
@@ -8,11 +8,11 @@
 import SwiftUI
 
 struct PlacesRow: View {
-    let feature: Feature
-    
+    let state: PlacesRowState
+    //Dotaz private
```

Comment:
ten state muze byt private ale nemusi, asi zalezi na preferenci programatora... pokud je private, tak je to o neco "čiščí", ale zase potom musis dopisovat init, protoze se ti nevygeneruje ten automaticky (slozeny pouze s public properties).